[examples/hackernews] Example is broken.

```
$ docker compose build
...
 => ERROR [reactive_cache 6/6] RUN npm run build                                                                                                                                                                                                                                                                          1.8s
------
 > [reactive_cache 6/6] RUN npm run build:
0.202 
0.202 > reactive-hackernews@1.0.0 build
0.202 > tsc
0.202 
1.803 src/hackernews.service.ts(5,3): error TS2305: Module '"@skipruntime/api"' has no exported member 'Values'.
1.803 src/hackernews.service.ts(84,17): error TS7006: Parameter 'a' implicitly has an 'any' type.
1.803 src/hackernews.service.ts(84,20): error TS7006: Parameter 'b' implicitly has an 'any' type.
1.803 src/hackernews.service.ts(85,23): error TS7006: Parameter 'p' implicitly has an 'any' type.
------
failed to solve: process "/bin/sh -c npm run build" did not complete successfully: exit code: 2
```

--- a/examples/hackernews/reactive_service/package.json
+++ b/examples/hackernews/reactive_service/package.json
@@ -10,7 +10,7 @@
   "description": "",
   "dependencies": {
     "postgres": "^3.4.5",
-    "@skipruntime/server": "^0.0.4"
+    "@skipruntime/server": "^0.0.7"
   },
   "devDependencies": {
     "@skiplabs/eslint-config": "^0.0.1",

--- a/skipruntime-ts/addon/package.json
+++ b/skipruntime-ts/addon/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@skipruntime/addon",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "gypfile": true,
   "type": "module",
   "exports": {
@@ -19,6 +19,6 @@
     "mocha": "^11.0.1"
   },
   "dependencies": {
-    "@skipruntime/core": "^0.0.3"
+    "@skipruntime/core": "^0.0.4"
   }
 }

--- a/skipruntime-ts/core/package.json
+++ b/skipruntime-ts/core/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@skipruntime/core",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "type": "module",
   "exports": {
     ".": "./dist/index.js",

--- a/skipruntime-ts/examples/package.json
+++ b/skipruntime-ts/examples/package.json
@@ -12,8 +12,8 @@
     "@types/better-sqlite3": "^7.6.12"
   },
   "dependencies": {
-    "@skipruntime/helpers": "^0.0.6",
-    "@skipruntime/server": "^0.0.6",
+    "@skipruntime/helpers": "^0.0.7",
+    "@skipruntime/server": "^0.0.7",
     "eventsource": "^2.0.2",
     "better-sqlite3": "^11.7.2"
   }

--- a/skipruntime-ts/helpers/package.json
+++ b/skipruntime-ts/helpers/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@skipruntime/helpers",
-  "version": "0.0.6",
+  "version": "0.0.7",
   "type": "module",
   "exports": {
     ".": "./dist/index.js",
@@ -21,7 +21,7 @@
   "dependencies": {
     "eventsource": "^2.0.2",
     "express": "^4.21.1",
-    "@skipruntime/core": "^0.0.3",
+    "@skipruntime/core": "^0.0.4",
     "@skipruntime/api": "^0.0.5"
   },
   "devDependencies": {

--- a/skipruntime-ts/runtime/package.json
+++ b/skipruntime-ts/runtime/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@skipruntime/runtime",
-  "version": "0.0.2",
+  "version": "0.0.3",
   "type": "module",
   "exports": {
     ".": "./dist/index.js"
@@ -11,9 +11,9 @@
     "lint": "eslint"
   },
   "dependencies": {
-    "@skipruntime/wasm": "^0.0.4"
+    "@skipruntime/wasm": "^0.0.5"
   },
   "optionalDependencies": {
-    "@skipruntime/addon": "^0.0.2"
+    "@skipruntime/addon": "^0.0.3"
   }
 }

--- a/skipruntime-ts/server/package.json
+++ b/skipruntime-ts/server/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@skipruntime/server",
-  "version": "0.0.6",
+  "version": "0.0.7",
   "type": "module",
   "exports": {
     ".": "./dist/server.js"
@@ -15,6 +15,6 @@
   },
   "dependencies": {
     "express": "^4.21.1",
-    "@skipruntime/runtime": "^0.0.2"
+    "@skipruntime/runtime": "^0.0.3"
   }
 }

--- a/skipruntime-ts/tests/package.json
+++ b/skipruntime-ts/tests/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@skipruntime/tests",
-  "version": "0.0.7",
+  "version": "0.0.8",
   "type": "module",
   "exports": {
     ".": "./dist/tests.js"
@@ -12,7 +12,7 @@
   },
   "dependencies": {
     "earl": "^1.3.0",
-    "@skipruntime/core": "^0.0.3",
-    "@skipruntime/helpers": "^0.0.6"
+    "@skipruntime/core": "^0.0.4",
+    "@skipruntime/helpers": "^0.0.7"
   }
 }

--- a/skipruntime-ts/wasm/package.json
+++ b/skipruntime-ts/wasm/package.json
@@ -1,6 +1,6 @@
 {
   "name": "@skipruntime/wasm",
-  "version": "0.0.4",
+  "version": "0.0.5",
   "type": "module",
   "exports": {
     ".": "./dist/src/skip-runtime.js"
@@ -17,7 +17,7 @@
     "mocha": "^11.0.1"
   },
   "dependencies": {
-    "@skipruntime/core": "^0.0.3",
+    "@skipruntime/core": "^0.0.4",
     "@skip-wasm/json": "^1.0.5"
   }
 }

